src/components/Sections/SectionChart/SectionBarChart.js

```diff
@@ -240,18 +240,10 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
                   }}
                   label={!!chartProperties.label}
                 >
-                  {!chartProperties.label && item.showValues &&
+                  {chartProperties.showValues && i === dataItems.length - 1 &&
                   <LabelList
                     position="top"
-                    valueAccessor={(entry) => {
-                      let value = '';
-                      if (entry && entry.data && entry.data[0] !== undefined) {
-                        value = entry.data[0];
-                      } else if (entry && entry.value && entry.value[1] !== undefined) {
-                        value = entry.value[1];
-                      }
-                      return value;
-                    }}
+                    dataKey={group => group.total}
```

Comment:
i think this should to the job 
```suggestion
                    dataKey={group => group.data[0}
```

Reply:
i need the total, so i rather make sure its calculated